root/static/scripts/edit/URLCleanup.js

```diff
@@ -2751,7 +2751,7 @@ const CLEANUPS = {
         return {
           error: exp.l(
             `Links to specific sections of Wikipedia articles are not 
-             allowed. Please remove “{fragment}” if still appropriate.
+             valid. Please remove “{fragment}” if still appropriate.
```

Comment:
I dunno. These are valid links, we just don't allow them.

Reply:
Your point is valid. :wink: I felt something was amiss, but I was probably just missing the link to the guidelines at the very end of the message.

Reply:
Dropped that change too.